static/app/views/dashboards/widgets/timeSeriesWidget/timeSeriesWidgetVisualization.stories.tsx

```
@@ -471,7 +471,7 @@ export default storyBook('TimeSeriesWidgetVisualization', (story, APIReference)
 
   story('Legends', () => {
     const [legendSelection, setLegendSelection] = useState<LegendSelection>({
-      'p99(span.duration)': false,
```

why did this get changed eye👀

---

the series in the chart is called `p99`, so having `p99(span.duration)` in the state by default did nothing to hide data.

Now when the story loads you can see the json printed `{p99: false}` while the chart has 2 series in the legend, but only one series is rendered.

I think it's more obvious this way that the `legendSelection` prop is working properly without having to click first.